gensrc/proto/types.proto

```diff
@@ -172,8 +172,15 @@ message IsNullPredicatePB {
 }
 
 message IndexValueWithVerPB {
-    repeated int64 versions = 1;
-    repeated uint64 values = 2;
+    // will be set after mvcc support
```

Comment:
will it cause any problem if upgrading from previous version?

looks like this protobuf change is not compatible at all.

Reply:
Cloud native index is not released yet, I need to merge this PR as soon as possible 😆

Reply:
ok